explorer/src/components/Account/AccountBalanceStats.tsx

```diff
@@ -40,14 +40,14 @@ export const AccountBalanceStats: FC<Props> = ({ account, isDesktop = false }) =
       <div className='flex w-full items-center lg:gap-4 lg:py-8'>
         <div className='flex flex-row justify-center gap-8 lg:flex-none lg:flex-col'>
           <div className='flex items-center'>
-            <div className='bg-purpleElectric mr-2 h-[30px] w-1' />
+            <div className='mr-2 h-[30px] w-1 bg-purpleElectric' />
```

Comment:
I'm just curious why these `classNames` are often changing order?

Reply:
Most colours used to be hard-coded so like bg-purpleElectric was bg-[#E970F8] now that these were changed for tailwind theme extended colours, I'm guessing prettier give them higher priority...